rootfs/etc/nginx/lua/balancer.lua

```diff
@@ -185,16 +185,26 @@ local function route_to_alternative_balancer(balancer)
   local target_header = util.replace_special_char(traffic_shaping_policy.header,
                                                   "-", "_")
   local header = ngx.var["http_" .. target_header]
+  local match_strategy = "exact"
+  if traffic_shaping_policy.matchStrategy and #traffic_shaping_policy.matchStrategy > 0 then
+    match_strategy = traffic_shaping_policy.matchStrategy
+  end
```

Comment:
I'd move this to `if traffic_shaping_policy.headerValue` clause since this is used only there.

Reply:
done